Assets/MixedRealityToolkit.SDK/Experimental/Features/UX/BoundsControl/Visuals/Configuration/BoundsControlLinksConfiguration.cs

```diff
@@ -0,0 +1,115 @@
+﻿
+using Microsoft.MixedReality.Toolkit.UI.Experimental.BoundsControlTypes;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Microsoft.MixedReality.Toolkit.UI.Experimental
+{
+    /// <summary>
+    /// Configuration for <see cref="BoundsControlLinks"/> used in <see cref="BoundsControl"/>
+    /// This class provides all data members needed to create a link of a bounds control
+    /// </summary>
+    [CreateAssetMenu(fileName = "BoundsControlLinksConfiguration", menuName = "Mixed Reality Toolkit/Bounds Control/Links Configuration")]
+    public class BoundsControlLinksConfiguration : ScriptableObject
```

Comment:
suggestion, feel free to ignore.

What if we add Microsoft.MixedReality.Toolkit.UI.Experimental.BoundsCOntrol namespace, in this way we could remove "BoundsControl" from all class names?

Not a big deal either way, just a consideration to make things more readable

Reply:
it's a good suggestion. i had this a while ago when i had fewer classes but thought it's harder to find them that way in the asset explorer. now that the number of classes has grown i think it makes sense to do that.